_build_cfg.yml

```diff
@@ -167,8 +167,13 @@ Topics:
     File: high_availability
   - Name: Self-Provisioned Projects
     File: selfprovisioned_projects
-  - Name: Persistent Storage Using NFS
-    File: persistent_storage_nfs
+  - Name: Persistent Storage
+    Dir: persistent_storage
+    Topics:
+      - Name: Persistent Storage Using NFS
+        File: persistent_storage_nfs
+      - Name: Persistent Storage Using GlusterFS
+        File: persistent_storage_glusterfs
```

Comment:
Since the `Topics.Name` fields here can show a different title in the left nav than what appears in the actual `.adoc` files' title heading, how about for the left nav just have "Using NFS" and "Using GlusterFS", respectively, since it's a bit redundant w/ the "Persistent Storage" directory heading already right above it. And the title headings in the `adoc` files themselves could stay as-is (e.g., "Persistent Storage Using GlusterFS", etc).